utility/inc/logger/loggeriface.hpp

```diff
@@ -34,28 +35,17 @@ class LoggerInterface {
     * Answer: https://stackoverflow.com/a/27137475
     */
     template <typename Duration>
-    void printTime(tm t, Duration fraction) {
-        using namespace std::chrono;
-        std::printf("[%04u-%02u-%02u %02u:%02u:%02u.%03u]", t.tm_year + 1900,
+    std::string printTime(tm t, Duration fraction) {
```

Comment:
this function is no longer printing the timestamp.

should rename this formatTimestamp()